### PR TITLE
Creating a stub for fuquay varina

### PR DESCRIPTION
**Pull request by githubteacher**
This file adds basic headings for the hometown file for GitHubTeacher.

### CODE REVIEW

File: fuquay-varina.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Fuquay Varina, NC
+
+### Places to eat
+
```
Comment (githubteacher) [inline fuquay-varina.md:4]: @crichID I really like hot dogs, can you recommend a good place to get one in Fuquay?

**Comment by githubteacher** [comment]
@halfaro1 would you mind looking at this and let me know if it looks good to :ship:

**Comment by hectorsector** [comment]
:shipit:

**Comment by githubteacher** [comment]
This PR closes #4

**Pull request merged**